Description: Enable generation of man page and completions for Debian
Forwarded: not-needed
Last-Update: 2025-01-09
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -302,2 +302,3 @@
 default = [
+    "man-completion-gen",
     #"online-tests",
